templates/launch-config.yaml.tpl

```diff
@@ -0,0 +1,12 @@
+queues: [${queues}]
+entity: ${entity}
+max_jobs: ${max_jobs}
+max_schedulers: ${max_schedulers}
+environment:
+  type: azure
+registry:
+  type: acr
+  uri: ${uri}
+builder:
+  type: kaniko
+  build-context-store: ${build_context_store_uri}
```

Comment:
whats this for?

Reply:
This is the launch agent config file that get's passes to the helm chart https://docs.wandb.ai/guides/launch/run-agent#agent-configuration